tests/snapshots/tests.contrib.aiohttp.test_aiohttp_client.test_200_request.json

```diff
@@ -0,0 +1,46 @@
+[[
+  {
+    "name": "aiohttp.request",
+    "service": null,
+    "resource": "aiohttp.request",
+    "trace_id": 0,
+    "span_id": 1,
+    "parent_id": 0,
+    "type": "http",
+    "meta": {
+      "http.method": "GET",
+      "http.status_code": "200",
+      "http.status_msg": "OK",
+      "http.url": "http://localhost:8001/status/200",
+      "runtime-id": "6ef1d2e57c6c46858c8a2ae5b4b815db"
+    },
+    "metrics": {
+      "_dd.agent_psr": 1.0,
+      "_dd.top_level": 1,
+      "_dd.tracer_kr": 1.0,
+      "_sampling_priority_v1": 1,
+      "system.pid": 54671
+    },
+    "duration": 24047000,
+    "start": 1646300282382895000
+  },
+     {
+       "name": "TCPConnector.connect",
+       "service": null,
+       "resource": "TCPConnector.connect",
+       "trace_id": 0,
+       "span_id": 2,
+       "parent_id": 1,
+       "duration": 1728000,
+       "start": 1646300282383136000
+     },
+        {
+          "name": "TCPConnector._create_connection",
```

Comment:
do we need to trace these both if they are nested?

Reply:
I actually removed this instrumentation but didn't update the snapshot here. It will be a follow-up task for whether or not we want to instrument the connection layer.